components/brave_wallet/browser/unstoppable_domains_dns_resolve.cc

```diff
@@ -11,23 +11,25 @@ namespace brave_wallet::unstoppable_domains {
 
 namespace {
 enum RecordKeys {
-  DWEB_IPFS_HASH,
-  IPFS_HTML_VALUE,
-  DNS_A,
-  DNS_AAAA,
-  BROWSER_REDIRECT_URL,
-  IPFS_REDIRECT_DOMAIN_VALUE,
-  RECORD_KEY_COUNT,
+  kDwebIpfsHash,
+  kIpfsHtmlValue,
+  kDnsA,
+  kDnsAAAA,
+  kBrowserRedirectUrl,
+  kIpfsRedirectValue,
+  kKeyCount,
 };
 
 // See
 // https://docs.unstoppabledomains.com/developer-toolkit/resolve-domains-browser/browser-resolution-algorithm/
 // for more details.
-const constexpr char* const kRecordKeys[] = {
+constexpr const char* kRecordKeys[] = {
     "dweb.ipfs.hash", "ipfs.html.value",      "dns.A",
-    "dns.AAAA",       "browser.redirect_url", "ipfs.redirect_domain.value"};
-static_assert(static_cast<size_t>(RecordKeys::RECORD_KEY_COUNT) ==
-                  sizeof(kRecordKeys) / sizeof(kRecordKeys[0]),
+    "dns.AAAA",       "browser.redirect_url", "ipfs.redirect_domain.value",
```

Comment:
huh, I was expecting that it will become a one row thing :)

Reply:
It seems it doesn't work like that :D